addBag.php

```diff
@@ -19,26 +25,40 @@
         <form method ="post">
             <div>
                 <label>Brand: </label>
-                <input type="text" name="brand" required/><br>
+                <input type="text" name="brand" /><br>
             </div>
             <div>
                 <label>Model: </label>
-                <input type="text" name ="model" required/><br>
+                <input type="text" name ="model" /><br>
             </div>
             <div>
                 <label>Price (£): </label>
-                <input type="text" name ="price" required/><br>
+                <input type="text" name ="price" /><br>
             </div>
             <div>
-                <label>Materials (optional): </label>
-                <input type="text" name ="materials"/><br>
+                <label>Materials: </label>
```

Comment:
I'm not sure it matters a great deal, but all of the examples I've seen of label tags associate them to a form using label for="fieldname" where fieldname is the name of the input. In this case for example it would be for="materials"